courses/utils.py

```diff
@@ -173,3 +173,21 @@ def get_unenrollable_courses(queryset):
         .filter(courseruns__id__in=courseruns_qs.values_list("id", flat=True))
         .distinct()
     )
+
+
+def get_archived_courseruns(queryset):
```

Comment:
Could we add a test for this new function?

Reply:
@annagav Do you think we could consider merging this as is, and do the test as a follow-up? It would be great to see this availability update come through on MIT Learn.